scripts/pr-ci-matrix.rb

```diff
@@ -47,6 +47,7 @@ def minimum_version(major)
   'cocoapods-ios-dynamic' => oldest_and_latest,
   'cocoapods-watchos' => oldest_and_latest,
   # 'cocoapods-catalyst' => oldest_and_latest,
+  'swiftui' => latest,
```

Comment:
You also need to run this script to populate the file that jenkins actually looks at.